### PR TITLE
Add autocomplete

### PR DESCRIPTION
**Pull request by loverajoel**
#### What does this PR do?

Add autocomplete
image

### CODE REVIEW

File: src/components/search/Search.js [hunk 1]
```diff
@@ -1,18 +1,68 @@
...
 
 class Search extends Component {
+    _handleKeyPress(event) {
+        if (event.key === 'Enter') {
+            const text = event.target.value;
+            if (text.length > 3) {
+                this._search(text);
+            }
+        }
+    }
```
Comment (Cosmitar) [inline src/components/search/Search.js]: Why was _handleKeyPress moved from below render() to above? Not sure if it is a best practice but I think I've read a suggestion about custom methods should be put below native ones.

**Comment by Cosmitar** [comment]
:shipit: :+1: welcome back!

File: src/components/search/Search.js (hunk 1 above)
Comment (loverajoel) [inline src/components/search/Search.js]: yep, I'm going to change

**Comment by loverajoel** [comment]
@Cosmitar done

**Pull request merged**